parser/parser.y

```diff
@@ -5221,6 +5230,53 @@ AdminStmt:
 			JobIDs: $6.([]int64),
 		}
 	}
+|	"ADMIN" "SHOW" "LOG" AdminShowLog
+	{
+		$$ = &ast.AdminStmt{
+			Tp: ast.AdminShowLog,
+			ShowLog: $4.(*ast.ShowLog),
+		}
+	}
+
+AdminShowLog:
+	"RECENT" NUM
```

Comment:
We can remove the `RECENT` and make the `NUM` optional

Reply:
There will be two containers, one heap for TopN , one queue for Recent.
Suggested by @shenli

Reply:
No "TOP" means recent.

Reply:
TOP doesn't mean `recent`. It means `order by x limit n`.